tasks/fr/insee.py

```diff
@@ -277,13 +277,28 @@ def columns(self):
             type='Text',
             name='DCOMIRIS',
             description='Full Code IRIS. Result of the concatenation of DEPCOM and IRIS attributes. ',
-            weight=0,
+            weight=1,
```

Comment:
Also, not sure why INSEE changes are leaking into the canary islands fix.  Can you remove these changes and add them to the correct branch? Looks like changes to La Caixa leaked in here, too.